tests/queries/tests.py

```diff
@@ -3086,20 +3099,10 @@ def test_iter_exceptions(self):
         with self.assertRaisesMessage(AttributeError, msg):
             list(qs)
 
-    def test_invalid_qs_list(self):
-        # Test for #19895 - second iteration over invalid queryset
-        # raises errors.
-        qs = Article.objects.order_by('invalid_column')
-        msg = "Cannot resolve keyword 'invalid_column' into field."
-        with self.assertRaisesMessage(FieldError, msg):
-            list(qs)
-        with self.assertRaisesMessage(FieldError, msg):
-            list(qs)
-
```

Comment:
This test was now redundant; `FieldError` is raised at `order_by` call time and not at query compilation time anymore which is covered by the following test.